demo/admin/src/products/ManufacturerForm.tsx

```diff
@@ -227,54 +232,81 @@ export function ManufacturerForm({ id }: FormProps): React.ReactElement {
                                 name="address.country"
                                 label={<FormattedMessage id="manufacturer.address.country" defaultMessage="Address Country" />}
                             />
-                            <FieldSet
-                                collapsible={true}
-                                initiallyExpanded={false}
-                                title={<FormattedMessage id="manufacturer.address.alternativeAddress" defaultMessage="Alt-Address" />}
-                                supportText={
-                                    <FormattedMessage
-                                        id="manufacturer.address.alternativeAddress.supportText"
-                                        defaultMessage="An alt-address to be used"
+                            <Divider sx={{ marginBottom: 5 }} />
+                            <Field
+                                fullWidth
+                                name="useAlternativeAddress"
+                                type="checkbox"
+                                label={<FormattedMessage id="manufacturer.address.useAlternativeAddress" defaultMessage="Use alternative address" />}
+                            >
+                                {(props) => (
+                                    <FormControlLabel
+                                        label={
+                                            props.input.checked ? (
+                                                <FormattedMessage id="manufacturer.address.useAlternativeAddress.yes" defaultMessage="Yes" />
+                                            ) : (
+                                                <FormattedMessage id="manufacturer.address.useAlternativeAddress.no" defaultMessage="No" />
+                                            )
```

Comment:
We have generic messages in `@comet/admin` for this: https://github.com/vivid-planet/comet/blob/next/packages/admin/admin/src/messages.ts#L31-L32